### PR TITLE
fixes #4256  Links don't wrap

### LINKED ISSUE #4256: [USERSNAP] Links don't wrap
**Taken at**: https://pioneerapp.xyz/#/forum/thread/308
**Fields**:
- Houston, we have a problem.: Links don&#39;t wrap
- Comment: links should wrap, just like other text.

SCREENSHOT
*[Download original image](https://resources.usersnap.com/company/f9f719df-a929-4461-bb6e-46dbfbceffb7/datapoint_screenshot/666a62c8-e2de-4d94-a47b-ccd9091375ce-annotated_c7a12db1-16fa-4204-9805-be5feafe8599.png?etag=88e8999654f3a450f843cfed2002d892)*

**Browser**: Chrome 111 (Windows 10)
**Screen size**: 3840x2160
**Browser size**: 1366x657
**[Open #405 in Usersnap](https://app.usersnap.com/l/feedback/5e7e15ed-5e6a-44e9-8917-428cd9bb99e7?utm_medium=integration&utm_campaign=integration&utm_content=open)**

Powered by **[Usersnap](https://usersnap.com/?utm_source=product&amp;utm_medium=poweredbylink&amp;utm_campaign=github_entry)**.

### PR DESCRIPTION
**Pull request by IlyaSmiyukha**
- #4048

### CODE REVIEW

**Comment by IlyaSmiyukha** [comment]
@chrlschwb Check pls

**Review by chrlschwb — APPROVED** [review]

**Comment by chrlschwb** [comment]
Please contact me on discord. My discord handle: @Chaos77

**Review by thesan — CHANGES_REQUESTED** [review]
The issue only happens on Chrome. But this PR doesn't solve it for me:
image

Tested on: https://dao-git-fork-ilyasmiyukha-4256-joystream.vercel.app/#/forum/thread/324

**Comment by IlyaSmiyukha** [comment]
Missed fix for order list. Should be fine now

File: packages/ui/src/common/components/MarkdownPreview/MarkdownPreviewStyles.tsx [hunk 1]
```diff
@@ -88,18 +88,19 @@ export const MarkdownPreviewStyles = createGlobalStyle<MarkdownPreviewStylesProp
...
-    grid-row-gap: 8px;
     width: 100%;
     margin: 0;
     padding-left: 24px;
     list-style: none;
   }
 
   .markdown-preview ol {
-    display: grid;
     grid-row-gap: 8px;
```
Comment (thesan) [inline packages/ui/src/common/components/MarkdownPreview/MarkdownPreviewStyles.tsx:104]: Thank you! Sorry I didn't notice you had fixed it for `ul` only.

Just one small detail:
```suggestion
```

**Review by thesan — CHANGES_REQUESTED** [review]

**Review by thesan — APPROVED** [review]
Perfect, thanks !

**Pull request merged**